### ISSUE TITLE
The droprate seems bugged in some cases

### ISSUE THREAD

**Issue by Neuroncz** [TeamEverlook/Everlook-BugTracker#267]
I repeatedly farm BRD and noticed that items with a small drop appear much more often than they should. I believe that this problem does not only concern of the BRD instance, but also the world drops and others, considering how the auction is overwhelmed with some items.

https://classicdb.ch/?item=18654
https://classicdb.ch/?item=18661
https://classicdb.ch/?item=16048
https://classicdb.ch/?item=16049

Some items maybe have less drop than they should have from my experience. (Warder Stilgiss - Arbiter's Blade)

https://classicdb.ch/?item=11784-0

It would be good to check these divergentions, especially for rare and epic items and last but not least also for materials such as pearls and diamonds, If the server is to have 1x drop rate.

At this moment I have some doubts about whether the server is really Blizzlike.

**Comment by red-candle** [comment]
This server is not blizzlike, it is **custom**:

- Repeatable Rend quest (added on purpose)
- Deserter when declining AV queue (added on purpose)
- Limited talent respec cost (added on purpose)
- Talent respec fade/reset over time before 1.11 (added on purpose)
- Extra line of sight checks that never existed in vanilla
- Stratholme (live) mobs drop on average one BoE epic on each run
- High weapon skill gain rate
- 5-minute mail delivery to characters on other accounts, instead of 1h
- Instant money delivery for AH sales, instead of 1h
- Rogue improved exposed armor that adds another 50% to all physical damage on top of the extra armor reduction, killing PvP and raid bosses
- Alterac Valley queues hidden in sign up list so that people can't avoid certain instances (added on purpose)

**Comment by Neuroncz** [comment]
Not true, web claims 1x exp/1x drop rates. PvP server. 10k average peak population.
There will be no geo-bans on the server -- this means that anyone from all over the world is welcome to play. You caan read it in baasic info. This is absurd! There is no mention of a custom server here, nobody even cares about a custom server!

**Comment by Neuroncz** [comment]
This problem wasn't fixed yet. 
1. These are not custom drops
2. I absolutely do not agree with the custom server
